middleware.js

```diff
@@ -108,6 +110,12 @@ module.exports = function(options) {
         sassDir = dirname(sassPath);
       }
 
+      if (sourceMap === true) {
+        var cssMapPath = cssPath + '.map';
+      } else if (typeof sourceMap === 'string') {
+        var cssMapPath = join(dirname(cssPath), sourceMap);
+      }
+
```

Comment:
These should be passed to node-sass as is. We are processing the options there: [index.js#L66L74](https://github.com/sass/node-sass/blob/146c8e096e1797f656ad81e980e88a363d2eaba4/lib/index.js#L66L74).